pkg/ibmvpc/examples/experiments_env_all_vpcs.json

```
@@ -0,0 +1,6078 @@
+{
+    "test-vpc1-ky": {
+        "endpoints_connectivity": [
+            {
+                "src": {
+                    "ResourceType": "Public Internet",
```

```suggestion
{
    "test-vpc1-ky": {
        "endpoints_connectivity": [
            {
                "src": {
                    "ResourceType": "Public Internet",
```
```suggestion
[
    {
        "vpc_name": "test-vpc1-ky",
        "endpoints_connectivity": [
            {
                "src": {
                    "ResourceType": "Public Internet",
```
Would something like this be better (in case we would like to provide more VPC details in the future)? What do you think?

---

I think it might be better for having the option to add more vpc details, though Karen asked for a map instead of list.. @kyorav  what do you think?

---

I am not following what exactly is the question. Keeping the info as a map does not preclude adding more keys to the object, including a "vpc_name" key, does it?
Anyway, the argument for making this into a map is extremely thin, if it's better as an array then go for it.